### DISCUSSION TITLE
Keeping Two Remote MobX Stores in Sync

### DISCUSSION THREAD

**Discussion by omranjamal** [mobxjs/mobx#3702] [Q&A]
Is there a way I could serialize a change-object upon making changes to a MobX store and send it over the wire to apply the change object to another remote store that has the exact same shape (and source code)?

**Comment by kubk** [comment]
While the exact solution will depend on your needs, I would suggest searching for inspiration in the following libraries:
- https://github.com/mobxjs/serializr
- https://github.com/quarrant/mobx-persist-store
